torch/_functorch/aot_autograd.py

```diff
@@ -3198,11 +3235,14 @@ def call_compiled_backward():
                     return tuple(out)
                 ctx.maybe_clear_saved_tensors()
                 if CompiledFunction.compiled_bw is None:
-                    context = torch._C._DisableAutocast if disable_amp else nullcontext
-                    with tracing(saved_context), context(), track_graph_compiling(aot_config, "backward"):
-                        CompiledFunction.compiled_bw = aot_config.bw_compiler(
-                            bw_module, placeholder_list
-                        )
+                    if CompiledFunction.bypass_compile_bwd and not ctx._is_compiled_autograd_tracing():
+                        CompiledFunction.compiled_bw = bw_module
```

Comment:
We probably need some sort of log message about when this situation shows up

Reply:
Huh - is this saying that when we saw some backward hooks but noticed that compiled autograd is enabled, we'll just keep running but silently not compile the backward (and instead, run the op-by-op aten backward graph directly?)

If that's the case, it seems like we should just error out in the forward as soon as we detect this happening instead (with an error message saying to turn on compiled autograd)